File: Two Pointers/Valid Palindromes/validPalindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string sLower;
        for(auto i: s){
            if (isalnum(i) != 0){
                sLower+=i;
            }
        }

        int fwd = 0;
        int back = sLower.length() - 1;

        for(int i = 0; i<= floor(sLower.length()/2); i++){
            if (sLower[fwd] != sLower[back]){
                return false;
            }
            else{
                fwd += 1;
                back -= 1;
            }
        }
        return true;
    }
};

File: Two Pointers/Valid Palindromes/validPalindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        sLower = ""
        for i in s:
            if i.isalpha() or i.isdigit():
                sLower += i
        sLower = sLower.lower()
        
        forPointer = 0
        backPointer = len(sLower) - 1

        for i in range(len(sLower)//2):
            if sLower[forPointer] != sLower[backPointer]:
                return False
            else:
                forPointer+=1
                backPointer-=1
        return True
                
